src/sage/graphs/generators/smallgraphs.py

```diff
@@ -4571,6 +4917,211 @@ def TutteGraph():
+
+    A near-bipartite graph is non-Pfaffian if and only if it contains one of
+    the graphs `K_{3, 3}`, `\Gamma_1` and `\Gamma_2` as an `S`-minor.
+
+    Norine and Thomas [NT2007]_ use the term ``Twinplex`` to describe one of
+    the 12-vertex cubic graphs, `\Gamma_1` and `\Gamma_2`, as defined by
+    Fischer and Little [FiLi2001]_. However, the figure in their paper that
+    supposedly provides embeddings for the graphs labeled Cubeplex and Twinplex
+    actually shows both embeddings corresponding to Fischer and Little's
+    `\Gamma_1`, which is the Cubeplex graph. Followingly, for
+    ``embedding='NT'``, we present a correct version of the Twinplex graph
+    with a slight modification of the embedding that is labeled as ``Twinplex``
+    in the paper of Norine and Thomas [NT2007]_.
+
+    PLOTTING:
+
+    Upon construction, the position dictionary is filled to override
+    the spring-layout algorithm. For different values of the parameter
+    ``embedding``, the Twinplex graph is displayed as it is mentioned in the
+    respective paper/ book. Note that for ``embedding='NT'``, a correct
+    embedding of the Twinplex graph is displayed with a minor modification to
+    the (incorrect) embedding shown in the paper [NT2007]_.
+
+    INPUT:
+
+    - ``embedding`` -- string (default: ``'LM'``)
+
+      - ``'LM'`` displays the embedding as shown for `\Gamma_2` by Lucchesi and
+        Murty [LM2024]_
+
+      - ``'FL'`` displays the embedding as shown for `\Gamma_2` by Fischer and
+        Little [FiLi2001]_
+
+      - ``'NT'`` displays the correct embedding with a minor modification to
+        the one shown as the (incorrect) ``Twinplex`` by Norine and Thomas
+        [NT2007]_
+
+      - ``'RST'`` displays the embedding as shown for the ``Twinplex`` by
+        Robertson, Seymour and Thomas [RST2019]_
+
+    OUTPUT:
+
+    - ``G`` -- the Twinplex graph; note that a :class:`ValueError` is returned
+      if ``embedding`` is none of ``'FT'``, ``'NT'``, ``'RST'`` or ``'LM'``
+
+    EXAMPLES:
+
+    Construct and show the Twinplex graph::
+
+        sage: g = graphs.TwinplexGraph()
+        sage: g.name()
+        'Twinplex Graph'
+        sage: g.order()
+        12
+        sage: g.size()
+        18
+        sage: g.girth()
+        5
+        sage: g.diameter()
+        3
+        sage: g.is_hamiltonian()
+        True
+        sage: g.crossing_number()
+        2
+        sage: g.show()                          # long time                             # needs sage.plot
+
+    TESTS:
+
+    Note that all four embeddings refer to the same graph, the Twinplex graph,
+    aka `\Gamma_2`::
+
+        sage: fl = graphs.TwinplexGraph(embedding='FL')
+        sage: nt = graphs.TwinplexGraph(embedding='NT')
+        sage: rst = graphs.TwinplexGraph(embedding='RST')
+        sage: lm = graphs.TwinplexGraph(embedding='LM')
+        sage: fl.is_isomorphic(nt) and fl.is_isomorphic(rst) and \
```

Comment:
We never use `\` in doctest (at least we avoid it. You can do:
```
sage: all(fl.is_isomorphic(g) for g in (nt, rst, lm))
True
```
or put everything on a single line